torch/csrc/autograd/python_variable.cpp

```diff
@@ -43,6 +43,8 @@ PyObject * THPVariable_Wrap(Variable var)
     Py_RETURN_NONE;
   }
 
+  THPUtils_assert(var.dim() > 0, "variable must have at least 1 dimension");
```

Comment:
I think throwing an exception would be better here. We don't always check if the return value is null.